web/packages/core/src/ruffle-player.ts

```diff
@@ -551,6 +552,9 @@ export class RufflePlayer extends HTMLElement {
             // `allowScriptAccess` can only be set in `options`.
             config.allowScriptAccess = options.allowScriptAccess;
 
+            if (config.allowSwfDownload) {
```

Comment:
What's the point of this `if`? If `config.allowSwfDownload` is falsy (e.g. `false` or `undefined`) then `this.allowSwfDownload` will not be initialized and remain `undefined`, that is falsy. The same would be achieved without this `if` as well.